### PR TITLE
remove commented code

### PR DESCRIPTION
**Pull request by wilzbach**
we probably shouldn't have this commented code in the master branch?

### CODE REVIEW

File: source/mir/sparse/blas/gemm.d [hunk 1]
```diff
@@ -197,11 +197,6 @@ unittest
          [-1.0, 9, 4, 8],
          [9.0, 8, 3, -2]];
 
-    // a * b ==
```
Comment (9il) [inline source/mir/sparse/blas/gemm.d:200]: This is comment for user

File: source/mir/internal/utility.d [hunk 2]
```diff
@@ -39,19 +39,3 @@ alias ConstData(S : T[], T) = const(T)[];
 
 enum isVector(V) = is(V : T1[], T1) || is(V : Slice!(1, T2*), T2);
 enum isMatrix(M) = is(M : Slice!(2, T*), T);
-
-//enum isConstVector(T) ==
```
Comment (9il) [inline source/mir/internal/utility.d:43]: Yes, this can be removed

File: source/mir/sparse/blas/dot.d [hunk 3]
```diff
@@ -83,8 +83,6 @@ D dot(
         }
         continue;
         L:
-        //import mir.internal.math: fmuladd;
```
Comment (9il) [inline source/mir/sparse/blas/dot.d:86]: yes, can be removed

**Comment by wilzbach** [comment]
> This is comment for user

Oh I was too late - sorry. Rebased!

**Pull request merged**